library/src/scripts/headers/mebox/pieces/MessagesContents.tsx

```diff
@@ -43,7 +43,7 @@ export class MessagesContents extends React.Component<IProps> {
                     <FrameHeaderWithAction title={title}>
                         <LinkAsButton
                             title={t("New Message")}
-                            to={"/messages/inbox"}
+                            to={"/messages/new"}
```

Comment:
It should be messages/add I think, I tried /new, it gives permission problem, while /add works fine